index.html

```diff
@@ -66,7 +66,159 @@ <h3>I'm a Software Developer</h3>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <button class="see-project">See Project</button>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <button class="see-project">See Project</button>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <button class="see-project">See Project</button>
+      </div>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="about-me">
+        <h2 class="about-me-heading">About me</h2>
+        <p class="about-me-note">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button class="get-my-resume">Get my resume</button>
+      </div>
+      <hr class="stack">
+      <ul class="stacks">
+        <li>
+         <h4 class="languages">Languages</h4>
+          <ul class="list">
+           <li>Javascript</li>
+           <li>Ruby</li>
+           <li>HTML</li>
+           <li>CSS</li>
+          </ul>
+       </li>  
+       <li>
+        <h4 class="frameworks">Frameworks</h4>
+         <ul class="list">
+         <li>React.js</li>
+         <li>Ruby on Rails</li>
+         <li>Rspec</li>
+         <li>Capybara</li>
+       </ul>
+      </li>  
+      <li>
+        <h4 class="skills">Skills</h4>
+         <ul class="list">
+           <li>Database Management</li>
```

Comment:
- [ ] Accorfimg to to the previouse reviewer  Kindly make sure to always follow the HTML rule which says that the only direct child of ul tag is li. It means no any other tag can be a direct child to ul. You have added ul tag as a direct child of ul which is wrong. Kindly make sure to add the list of tecnhlogies inside li tag. Kindly read [this article](https://www.freecodecamp.org/news/html-bullet-points-how-to-create-an-unordered-list-with-the-ul-tag-example/#:~:text=You%20can%20create%20a%20nested,the%20ul%20tag%20is%20li%20.&text=You%20create%20the%20nested%20unordered,list's%20item%20of%20your%20choosing.) for more info.Please kindly consider the technology categories ("Languages", "Frameworks" and "Skills") are in a ul , each of them in a li. Also, make sure the technology in each category (e.g., "JavasScript", "HTML" and "CSS") is also a list ul and each of them a li.

according to the project requirement see screenshot below 
![ul item](https://user-images.githubusercontent.com/92458236/235308359-e731ae8c-2c75-4fe8-ace1-dbb601633518.jpg) and to make it more clear see below example

>     
    <ul>
    <li>Language</li>
      <ul>
        <li>JavaScript</li>
        <li>Ruby</li>
        <li>HTML</li>
        <li>CSS</li>
      </ul>
      <li>Frameworks</li>
       </ul>